va/va_dec_av1.h

```diff
@@ -0,0 +1,546 @@
+    {
+        struct
+        {
+            /** \brief delta_q parameters
+             */
+            uint32_t        delta_q_present_flag                        : 1;
+            uint32_t        log2_delta_q_res                            : 2;
+
+            /** \brief delta_lf parameters
+             */
+            uint32_t        delta_lf_present_flag                       : 1;
+            uint32_t        log2_delta_lf_res                           : 2;
+
+            /** \brief CONFIG_LOOPFILTER_LEVEL
+             */
+            uint32_t        delta_lf_multi                              : 1;
+
+            /** \brief read_tx_mode
+             *  value range [0..6]
+             */
+            uint32_t        tx_mode                                     : 2;
+
+            /** \brief read_frame_reference_mode
+             */
+            uint32_t        reference_mode                              : 2;
+
+            uint32_t        reduced_tx_set_used                         : 1;
+
+            uint32_t        skip_mode_present                           : 1;
+
+            uint32_t        reserved                                    : 19;
+        } bits;
+        uint32_t            value;
+    } mode_control_fields;
+
+    /** \brief CDEF parameters
+     */
+    /*  value range [0..3]  */
+    uint8_t                 cdef_damping_minus_3;
+    /*  value range [0..3]  */
+    uint8_t                 cdef_bits;
+    /*  value range [0..63]  */
+    uint8_t                 cdef_y_strengths[8];
+    /*  value range [0..63]  */
+    uint8_t                 cdef_uv_strengths[8];
+
+    /** \brief loop restoration parameters
+     */
+    union
+    {
+        struct
+        {
+            uint16_t        yframe_restoration_type                     : 2;
+            uint16_t        cbframe_restoration_type                    : 2;
+            uint16_t        crframe_restoration_type                    : 2;
+            uint16_t        lr_unit_shift                               : 2;
+            uint16_t        lr_uv_shift                                 : 1;
+            uint16_t        reserved                                    : 7;
+        } bits;
+        uint16_t            value;
+    } loop_restoration_fields;
+
+    /** \brief global motion
+     */
+    VAWarpedMotionParamsAV1 wm[7];
+
+    /**@}*/
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                va_reserved[VA_PADDING_MEDIUM];
+} VADecPictureParameterBufferAV1;
+
+
+/**
+ * \brief AV1 Bit Stream Parameter Buffer Structure
+ *
+ * This structure conveys parameters related to bit stream data and should be
+ * sent once per tile.
+ *
+ * Bit stream data buffer of VASliceDataBufferType is used
+ * to send the bitstream.
+ *
+ * Please note that host decoder is responsible to parse out the
+ * per tile information. And the bit stream in sent to driver in per
+ * tile granularity.
+ */
+typedef struct _VABitStreamParameterBufferAV1

```

Comment:
How about renaming BitStream with Slice because VASliceDataBufferType is also used to send the bit stream data?

Reply:
yes, VASliceDataBufferType is used to send the bit stream data, which is same as other codecs